### PR TITLE
added 7 files for homework review - week3-js2

### PR DESCRIPTION
**Pull request by TITHI1244**

### CODE REVIEW

File: javascript/javascript2/week3/warmup.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+}
+runAfterDelay(3, function() {
+    console.log("Logged after 3 seconds. 'log from task 7'");
+});
+
+/* Check if we have double clicked on the page. A double click is defined by two clicks within 0.5 seconds. 
+If a double click has been detected, log out the text: "double click!" */
+document.addEventListener("dblclick", function() {
+    console.log("double click");
+});
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/warmup.js:103]: It works but try to rewrite using `click `event and `setTimeout`.

File: javascript/javascript2/week3/fastest_presser.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+/* -What if a user starts pressing a key before the game is started, what should happen?
+Nothing happens, the divs are unclickable before starting the game.
+-What if the game is a draw? Are both winners? None winners? Maybe indicate to the user that is was a draw.
+Indicated.
+-What if no time was specified for the game?
+An alert message to remind the user about timing.
+-What if there were no key presses before the game ends? 
+Reminds the user that s/he needs to press any key, l/S. */
```
Comment (NayanaKamtekar) [inline javascript/javascript2/week3/fastest_presser.js:85]: Nice work!!

**Review by NayanaKamtekar — APPROVED** [review]
Good work!!

File: javascript/javascript2/week3/warmup.js (hunk 1 above)
Comment (TITHI1244) [reply javascript/javascript2/week3/warmup.js:103]: Ok. Will change that. Thanks.

File: javascript/javascript2/week3/fastest_presser.js (hunk 2 above)
Comment (TITHI1244) [reply javascript/javascript2/week3/fastest_presser.js:85]: Thank you for your review.

**Pull request merged**